cnutil/CMakeLists.txt

```diff
@@ -11,11 +11,13 @@ include_directories("${MONERO_DIR}/src")
 include_directories("${MONERO_DIR}/external/easylogging++")
 
 link_directories(
-    ${MONERO_DIR}/src/cryptonote_core
-    ${MONERO_DIR}/src/cryptonote_basic
-    ${MONERO_DIR}/src/crypto
-    ${MONERO_DIR}/src/common
-    ${MONERO_DIR}/src/ringct
+    ${MONERO_DIR}/build/release/src/cryptonote_core
+    ${MONERO_DIR}/build/release/src/cryptonote_basic
+    ${MONERO_DIR}/build/release/src/crypto
+    ${MONERO_DIR}/build/release/src/common
+    ${MONERO_DIR}/build/release/src/ringct
+    ${MONERO_DIR}/build/release/contrib/epee/src
+    ${MONERO_DIR}/build/release/external/easylogging++
```

Comment:
Rather then altering the path which will break the build for everyone that uses cmake in the source tree, use an environment variable such as `MONERO_BUILD`.